policy/custom/definitions/policyset/DefenderForCloud.bicep

```diff
@@ -141,8 +136,8 @@ resource ascAzureDefender 'Microsoft.Authorization/policySetDefinitions@2020-03-
         groupNames: [
           'EXTRA'
         ]
-        policyDefinitionId: extensionResourceId(customPolicyDefinitionMgScope, 'Microsoft.Authorization/policyDefinitions', 'ASC-Deploy-Defender-OSSDB')
-        policyDefinitionReferenceId: toLower(replace('Deploy Azure Defender for Open-source relational databases', ' ', '-'))
+        policyDefinitionId: '/providers/Microsoft.Authorization/policyDefinitions/44433aa3-7ec2-4002-93ea-65c65ff0310a'
+        policyDefinitionReferenceId: toLower(replace('Configure Azure Defender for open-source relational databases to be enabled', ' ', '-'))
```

Comment:
Should this be called Microsoft Defender for open-source relational databases per this link? https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-defender-for-cloud-expands-sql-protection-to-open/ba-p/2384972#:~:text=Microsoft%20Defender%20for%20open-source%20relational%20databases%20is%20now,making%20open-source%20databases%20and%20SQL%20Server%20common%20targets.